spark/src/main/scala/org/apache/spark/sql/delta/DeltaParquetFileFormat.scala

```diff
@@ -104,7 +102,7 @@ case class DeltaParquetFileFormat(
   override def isSplitable(
     sparkSession: SparkSession, options: Map[String, String], path: Path): Boolean = isSplittable
 
-  def hasDeletionVectorMap: Boolean = broadcastDvMap.isDefined && broadcastHadoopConf.isDefined
+  def hasBroadcastHadoopConf: Boolean = broadcastHadoopConf.isDefined
```

Comment:
I think you can get rid of `broadcastHadoopConf` and `hasBroadcastHadoopConf` altogether here.

The only use we have for it now is to pass it to `rowIndexFilter.createInstance` when creating the DV filter but we also have `hadoopConf: Configuration` around (passed as an argument to `buildReaderWithPartitionValues`) which I believe should be sufficient here: we don't need this configuration to be broadcast first for the purpose of loading DVs from storage

Reply:
@johanl-db No clue why we had it back in the days hmm

Reply:
@johanl-db Removing `broadcastHadoopConf` doesn't seem to work, I found this [error message](https://runbot-ci.cloud.databricks.com/build/Delta-OSS-Scala212-Pr/run/17985292)but I haven't investigated yet, changing back to using the `broadcastHadoopConf` makes the tests passed